LibertyDSNP/example-client
# Feature/store user data in session 177602937

Purpose
---------------
As a user I want my data to be persistent as long as I have the page open. I don't want to have to re-login every time I refresh.


Solution
---------------
By storing our user login information and state from the Redux state in session storage we can make our global state persistent.


Change summary
---------------
* Created session storage
* Added user profile and wallet to session storage
* Added reload option for wallet api
* fixed tests for login and App
* Added icon for current logged in wallet by login button


Steps to Verify
----------------
1. Redux state is stored in sessionStorage
2. Login to example client
3. Hard refresh page
4. Should stay logged in
5. Close page
6. Reopen. should be logged out.
